test/test_in_docker.sh

```diff
@@ -29,7 +42,7 @@ function deployConjur() {
   if [ $CONJUR_DEPLOYMENT == "dap" ]; then
       cmd="$cmd --dap"
   fi
-  runDockerCommand "cd kubernetes-conjur-deploy-$UNIQUE_TEST_ID && $cmd"
+  runDockerCommand "cd kubernetes-conjur-deploy-$UNIQUE_TEST_ID && DEBUG=true $cmd"
```

Comment:
remove if not relevant

Reply:
I think it is relevant so that we can automatically see conjur-deploy in debug mode from all the failures

Reply:
@sigalsax the debug level is not related to the way we deploy Conjur (oss/dap). Can you please add this change and create a PR for it? we can still use your tag in this project as we diverged from master but if it has the value let's have it in master too.

Thanks :)